react-native-libraries.json

```
@@ -13242,5 +13242,12 @@
     "ios": true,
     "android": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/expo/expo/tree/main/packages/expo-video",
+    "npmPkg": "expo-video",
+    "ios": true,
+    "android": true,
```

Hello @RebeccaWallanderSoftwerk, thanks for adding missing Expo library! 🙏 

Let's update the supported platforms list to include web and tvOS, see:
* https://docs.expo.dev/versions/latest/sdk/video/

```suggestion
    "android": true,
    "web": true,
    "tvos": true,
```